UnitsNet/CustomCode/Quantities/Length.extra.cs

```diff
@@ -236,5 +236,70 @@ public string ToString(IFormatProvider? cultureInfo)
             // So inches are rounded when converting from base units to feet/inches.
             return string.Format(cultureInfo, "{0:n0} {1} {2:n0} {3}", Feet, footUnit, Math.Round(Inches), inchUnit);
         }
+
+        /// <summary>
+        ///     Outputs feet and inches on the format: {feetValue}' - {inchesValueIntegral}[ {inchesValueFractional}]"
+        ///     The inches are rounded to the nearest fraction of the fractionDenominator argument and reduced over the greatest common divisor.
+        /// </summary>
```

Comment:
Missing `/` before `{inchesValueIntegral}`.
Should probably also give a short description that the fractional will be omitted if it is `1`.